command/command.go

```diff
@@ -28,16 +29,51 @@ import (
 
 var verbose = false
 
+type LogLevel string
+
+const (
+	INFO LogLevel = "INFO"
+	WARN LogLevel = "WARN"
+	ERROR LogLevel = "ERROR"
+)
+
 func printVerboseMessage(message string) {
 	if verbose {
 		fmt.Println(message)
 	}
+	logToFile(INFO, message)
 	logger := client.Config{}.Logger
 	if logger != nil {
 		logger.Debug(message)
 	}
```

Comment:
can call new logToFile method for this debug message from SDK

Reply:
We already had a discussion. This logger is coming from go config. Hence did not change it